llvm/lib/Target/WebAssembly/WebAssemblyISelLowering.cpp

```diff
@@ -3400,8 +3436,11 @@ WebAssemblyTargetLowering::PerformDAGCombine(SDNode *N,
     return performVectorTruncZeroCombine(N, DCI);
   case ISD::TRUNCATE:
     return performTruncateCombine(N, DCI);
-  case ISD::INTRINSIC_WO_CHAIN:
+  case ISD::INTRINSIC_WO_CHAIN: {
+    if (auto AnyTrueCombine = performAnyTrueCombine(N, DCI.DAG))
```

Comment:
Nit, auto is mainly only used when the type already appears in a `cast<Foo>` or the type is complex like an std::pair
```suggestion
    if (SDValue AnyTrueCombine = performAnyTrueCombine(N, DCI.DAG))
```